src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -1633,6 +1633,7 @@ void Assembler::andb(Address dst, Register src) {
 }
 
 void Assembler::andw(Register dst, Register src) {
+  emit_int8(0x66);
```

Comment:
This is operand-size override prefix which modifies the default operand size16bits.